bucket/influxdb.json

```diff
@@ -1,32 +1,34 @@
 {
-    "version": "1.8.10",
+    "version": "2.1.1",
     "description": "Scalable datastore for metrics, events, and real-time analytics.",
     "homepage": "https://www.influxdata.com/",
     "license": "MIT",
+    "notes": [
+        "InfluxDB v1 has been removed to 'Versions', use 'scoop install versions/influxdb1' to install it.",
```

Comment:
```suggestion
        "InfluxDB v1 has been moved to 'Versions' bucket, use 'scoop install versions/influxdb1' to install it.",
```

Reply:
Wow, another VSCode Copilot error...

Reply:
Copilot can autocomplete scoop manifests!? Damn.

Reply:
It COULD, and I'm wondering where the source is or what the logic is of its autocompletion. Its codes are sometimes better than mine, shit...